pkg/cmd/pipeline/describe.go

```diff
@@ -30,53 +30,53 @@ import (
 	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-const describeTemplate = `Name:	{{ .PipelineName }}
+const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .PipelineName }}
```

Comment:
Not that this has to do with this pr, but it seems we don't have `Namespace` at the top of the template like other describe templates.

Reply:
Agreed but i am too worries to send a change with all that flakeyness going on 😓

Reply:
@chmouel No worries! Let's do a follow up pr. I'll open an issue for this.